### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by Gynjn** [facebookresearch/map-anything#78]
Thanks for sharing your great work.

BTW, I have a question about dataset curation.

I found that in the dataset details, there are some dynamic scene datasets.

Is it because of the dataset volume and these make the model more powerful?
With some elegant region masked loss function?

Thanks in advance 

Best, gyeongjin.

**Comment by Nik-V9** [comment]
Hi, thanks for your interest! We treat dynamic datasets similar to MonST3R (https://monst3r-project.github.io/).

We find that these datasets help a lot with data diversity and enable camera pose estimation to work despite the presence of dynamic objects.

We will have a follow up work called Any4D out in a few weeks which is catered to tackle dynamic scene 4D reconstruction.
